Microsoft.Toolkit.Uwp.UI.Controls/DropShadowPanel/DropShadowPanel.Properties.cs

```diff
@@ -59,6 +61,60 @@ public partial class DropShadowPanel
             DependencyProperty.Register(nameof(ShadowOpacity), typeof(double), typeof(DropShadowPanel), new PropertyMetadata(1.0, OnShadowOpacityChanged));
 
         /// <summary>
+        /// Gets a value indicating whether the platform supports drop shadows.
+        /// </summary>
+        /// <remarks>
+        /// On platforms not supporting drop shadows, this control has no effect.
+        /// </remarks>
+        public static bool IsSupported =>
+            !DesignMode.DesignModeEnabled &&
+            ApiInformation.IsTypePresent("Windows.UI.Composition.DropShadow"); // SDK >= 14393
+
+        /// <summary>
+        /// Gets or sets the casting element.
+        /// </summary>
+        [Deprecated("This property has been replaced with the Content property of the control. It is no longer required to place content within the Element property.", DeprecationType.Deprecate, 1)]
+        public FrameworkElement CastingElement
+        {
+            get
+            {
+                return this.Content as FrameworkElement;
+            }
+
+            set
+            {
+                this.Content = value;
+
+                ConfigureShadowVisualForCastingElement();
```

Comment:
Shouldn't this line be in the Content change property, now that this property is deprecated and "inactive" ?